src/vtt-segment-loader.js

```diff
@@ -307,19 +291,12 @@ export default class VTTSegmentLoader extends SegmentLoader {
    * @private
    */
   parseVTTCues_(segmentInfo) {
-    let decoder;
-    let decodeBytesToString = false;
-
-    if (typeof window.TextDecoder === 'function') {
-      decoder = new window.TextDecoder('utf8');
-    } else {
-      decoder = window.WebVTT.StringDecoder();
-      decodeBytesToString = true;
-    }
-
+    const fakeDecoder = {
```

Comment:
This might make more sense as an inline lambda, and we should probably have a comment around why it is used.